modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/Flamingos2Header.scala

```diff
@@ -17,7 +17,7 @@ import scalaz.concurrent.Task
   * Created by jluhrs on 2/10/17.
   */
 
-class Flamingos2Header(hs: DhsClient, f2ObsReader: Flamingos2Header.ObsKeywordsReader, f2Reader: Flamingos2Header.InstKeywordsReader, tcsKeywordsReader: TcsKeywordsReader) extends Header {
+class Flamingos2Header(hs: DhsClient, f2ObsReader: Flamingos2Header.ObsKeywordsReader, tcsKeywordsReader: TcsKeywordsReader) extends Header {
```

Comment:
If this is reverted the code won't compile with the new params. I suggest to add them in another PR